pi/index.html

```diff
@@ -1,7 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    
+   
+    <!-- http-server to run code  -->
```

Comment:
Can you undo these changes since they are not related to the issue? Thanks!

Reply:
I am not clear on the changes to undo, do you mean I should return the code to its former state (add the style attribute back)

Reply:
I have undone the changes on line 4 & 5, Please can you confirm, thank you @TildaDares